SeaPublicWebsite.Data/Migrations/20220620125327_AddBackupObjectToPropertyData.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace SeaPublicWebsite.Data.Migrations
+{
+    public partial class AddUpdatesObjectToPropertyData : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<int>(
+                name: "BackupPropertyDataId",
+                table: "PropertyData",
+                type: "integer",
+                nullable: true);
+            
+            migrationBuilder.AddForeignKey(
+                name: "FK_PropertyData_PropertyData_BackupPropertyDataId",
+                table: "PropertyData",
+                column: "BackupPropertyDataId",
+                principalTable: "PropertyData",
+                principalColumn: "PropertyDataId");
```

Comment:
I originally tried an 'Updates' object. The reason if failed is because it would seriously require rethinking the logic in the questionFlowService (because it would need to take into account the update object first and then the normal one).

This approach was also more flexible as I only needed to update the post methods and the AnserSummary_Get method.